src/components/FormComponents/InputController/InputController.tsx

```diff
@@ -22,18 +20,28 @@ export const InputController = <T extends FieldValues>({
   maxLength,
   tooltip,
   InputProps,
+  defaultNumberValue = 1,
   ...textFieldProps
 }: InputControllerProps<T>) => {
   const { t } = useTranslation('app');
 
+  const getTextAdornment = (value: number) => {
+    if (!value) return null;
+
+    if (textAdornment) {
+      return <StyledBodyLarge>{t(textAdornment, { count: value })}</StyledBodyLarge>;
+    }
+  };
+
   return (
     <Controller
       name={name}
       control={control}
       render={({ field: { onChange, value }, fieldState: { error } }) => {
         const handleAddNumber = () => onChange(+value + 1);
         const handleDistractNumber = () => {
-          +value > 1 && onChange(+value - 1);
+          const minNumberVal = defaultNumberValue === 0 ? 0 : 1;
+          +value > minNumberVal && onChange(+value - 1);
```

Comment:
I'm curious, what if the `value` will have not `number` type and in which cases? Maybe we need to predefine this value in case of `number` type and other whatever types before you use after